src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/viewtakes/viewmodel/ViewTakesViewModel.kt

```diff
@@ -29,23 +32,33 @@ class ViewTakesViewModel : ViewModel() {
     private val takeRepository = injector.takeRepository
     private val pluginRepository = injector.pluginRepository
 
-    val contentProperty = find(ProjectEditorViewModel::class).activeContentProperty
-    private val projectProperty = find(ProjectHomeViewModel::class).selectedProjectProperty
-    private var chapterProperty = find(ProjectEditorViewModel::class).activeChildProperty
+    var activeProperty: Collection by property()
+    val activeProjectProperty = getProperty(ViewTakesViewModel::activeProperty)
+
+    var activeCollection: Collection by property()
+    var activeCollectionProperty = getProperty(ViewTakesViewModel::activeCollection)
+
+    var activeContent: Content by property()
+    val activeContentProperty = getProperty(ViewTakesViewModel::activeContent)
 
     val selectedTakeProperty = SimpleObjectProperty<Take>()
+    var isSelectedTake = SimpleBooleanProperty(false)
+
+    private var context: TakeContext by property(TakeContext.RECORD)
+    val contextProperty = getProperty(ViewTakesViewModel::context)
 
     val alternateTakes: ObservableList<Take> = FXCollections.observableList(mutableListOf())
 
-    var title: String by property("View Takes")
+    var title: String by property("Verse ")
```

Comment:
is this text that should be from the content label? is there supposed to be a space after verse?